tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_included_record_parameters_negative.bal

```diff
@@ -279,3 +279,38 @@ function testInvokingFunctionContainingIncludedRecordParamOfTypeOtherThanRecordO
     _ = obj.fn(f = {});
     _ = obj.fn2(f = {});
 }
+
+type Record5 record {
+};
+
+type Record6 record {
+};
+
+function fn5(*Record5 record5) {
+}
+
+function fn6(*Record5 record5, *Record6 record6) {
```

Comment:
Shall we add tests with one open and one closed also?

Reply:
Updated